pkg/settings/setting.go

```diff
@@ -238,3 +238,13 @@ func getMetadataConfig() string {
 	}
 	return string(ans)
 }
+
+// getSettingByID returns a setting that is stored with the given id. If none is found
+// an error is return
+func GetSettingByID(id string) (Setting, error) {
+	setting, ok := settings[id]
```

Comment:
As @dramich  mentioned on my PR:
```suggestion
	setting, ok := provider.Get(id)
```

Reply:
After recent changes `func GetSettingByID(id string)` is not needed any more, it can be removed.

Reply:
Lets remove it then.